pcdsdevices/spectrometer.py

```diff
@@ -288,3 +288,33 @@ class Mono(BaseInterface, GroupDevice):
     removed = False
     transmission = 1
     SUB_STATE = 'state'
+
+
+class TMO_Spectrometer(BaseInterface, GroupDevice):
```

Comment:
Small thing, but usually we don't have underscores in class names as per pep8: https://www.python.org/dev/peps/pep-0008/#class-names